readthedocs/projects/exceptions.py

```diff
@@ -1,15 +1,46 @@
 """Project exceptions"""
 
+from django.conf import settings
+from django.utils.translation import ugettext_noop as _
 
-class ProjectImportError (Exception):
+from readthedocs.doc_builder.exceptions import BuildEnvironmentError
 
-    """Failure to import a project from a repository."""
 
-    pass
+class ProjectConfigurationError(BuildEnvironmentError):
```

Comment:
I think this class is missing the `get_default_message` method.

Reply:
It should be inheriting this from `BuildEnvironmentException`